Boards/arduino_mega.board.json

```diff
@@ -1,8 +1,9 @@
 {
   "$schema": "./mfboard.schema.json",
   "AvrDudeSettings": {
+    "Attempts":  3,
```

Comment:
avrdude defaults to 10 attempts. If we want to keep exact behaviour with the prior release of MobiFlight we could make this 10 for all non-Nano boards. Seems excessive though, either the connection works or it fails. Setting this to 3 (or 2 or even 1) would make the non-Nano boards fail faster during the flashing process.